contributors/design-proposals/pod-preset.md

```diff
@@ -133,6 +133,21 @@ type PodPresetSpec struct {
     // Selector is a label query over a set of resources, in this case pods.
     // Required.
     Selector      unversioned.LabelSelector
+
+    // InitContainerSelector is a ContainerSelector to select init containers
+    // of a PodSpec for injecting information into.
+    // If not specified, it will default to EmptyContainerSelector which matches
+    // any init-container in the PodSpec.
+    // +optional
+    InitContainerSelector *ContainerSelector
+
+    // AppContainerSelector is a ContainerSelector to select app containers of a
```

Comment:
@liggitt app container refer to regular containers. Realized App prefix is not a good idea :).
So removed the prefix, now AppContainerSelector --> ContainerSelector.